triage/2022-08-02.md

```diff
@@ -0,0 +1,179 @@
+# 2022-08-02 Triage Log
+
+A good week! We had one regression from [#99123](https://github.com/rust-lang/rust/pull/99123)
+that could use further study, but we had a huge swath of improvements. The downward trend of the
+[performance graph summary](https://perf.rust-lang.org/?start=&end=&kind=percentfromfirst&stat=instructions%3Au)
```

Comment:
This link is not fixed to a range, so it'll be ok now but quickly become wrong. Worth fixing the end points?

Reply:
Great idea, fixed in https://github.com/rust-lang/rustc-perf/pull/1385/commits/7d3c5f7e95a12b8379d2a9f10451a2285e26a7ee